src/resolvers/user.ts

```diff
@@ -13,7 +13,7 @@ export const resolvers: Resolvers = {
     // returns the current User
     me: async (_, __, context: MyContext): Promise<User> => {
       if (isAuthorized(context?.token)) {
-        return await User.findByEmail('me resolver', context, context.token.email);
+        return await User.findById('me resolver', context, context.token.id);
```

Comment:
Updated the "me" query to findById rather than findByEmail, since data was disappearing on the user profile page when a user updated their email preference.